packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.styles.ts

```diff
@@ -291,20 +290,26 @@ export const getStyles = (props: IDetailsHeaderStyleProps): IDetailsHeaderStyles
       {
         color: colors.iconForegroundColor,
         opacity: 1,
-        paddingLeft: 8,
-        selectors: {
-          '&$sortIcon': {
-            paddingLeft: 4,
-            position: 'relative',
-            top: 1
-          }
-        }
+        paddingLeft: 8
       }
     ],
 
-    sortIcon: [],
+    sortIcon: [
+      {
+        paddingLeft: 4,
+        position: 'relative',
+        top: 1
+      }
+    ],
 
-    filterChevron: [classNames.filterChevron],
+    filterChevron: [
+      classNames.filterChevron,
+      {
+        color: colors.dropdownChevronForegroundColor,
+        paddingLeft: 4,
+        verticalAlign: 'middle'
+      }
+    ],
```

Comment:
It's a bit concerning these don't show up in snapshot tests. Would that be easy to fix?

Reply:
maybe :)

Reply:
Yes, updated with new snapshot